src/constants/zerops.go

```diff
@@ -91,7 +96,7 @@ func findFirstWritablePath(paths []pathReceiver) string {
 func checkPath(filePath string) (string, error) {
 	dir := path.Dir(filePath)
 
-	if err := os.MkdirAll(dir, 0775); err != nil {
+	if err := os.MkdirAll(dir, 0666); err != nil {
```

Comment:
To jsou divny prava ne? `rw-rw-rw-` kdyz mas na adresar `w` tak vetsinou mas i `x`. Co je ucel? 


`0755` vlastnik vse, ostatni cist a prochazet
`0777` vsichni vsechno

Reply:
New setup
- Directory - `0755`
- Log file - `0666`
- cli.data && wg conf file - `0600` because they contain a wg private key